### PR TITLE
[RPC Gateway] Launch to 1% of Polygon

### PR DESCRIPTION
**Pull request by rollingtumbling**
Revert "Remove unnecessary change"

This reverts commit 5ee66a1e131abee525f5e3ebfeaefed97f01d8a6.

Add test settings

### CODE REVIEW

**Comment by rollingtumbling** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/547?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#547** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/547?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#546** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/546?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#545** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/545?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rollingtumbling and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: bin/stacks/routing-lambda-stack.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export class RoutingLambdaStack extends cdk.NestedStack {
       // 11/8/23: URA currently calls the Routing API with a timeout of 10 seconds.
       // Set this lambda's timeout to be slightly lower to give them time to
       // log the response in the event of a failure on our end.
-      timeout: cdk.Duration.seconds(9),
+      timeout: cdk.Duration.seconds(19),
```
Comment (jsy1218) [inline bin/stacks/routing-lambda-stack.ts:103]: let's not merge this.

I do see our cold start issue is starting to affect our local dev experience, as well as beta integ-tests. I actually spent some time investigating, and found out that the cold start is due to the routing-api S2 pool caching per uniswap subgraph version. Recently with more chains support, this does affect routing-api cold start.

I can log a ticket sometime today to describe the issue.

Comment (rollingtumbling) [reply bin/stacks/routing-lambda-stack.ts]: Sorry that's for debug only. Reverted that change.

**Comment by rollingtumbling** [comment]
We decide to postpone launch of Polygon. Instead, we will finish launch on other low revenue-generating chains first. @cgkol

**Pull request closed without merging**